package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "docs",
+  "scripts": {
+    "lint": "yarn install && yarn lint-md && yarn lint-js && yarn lint-php && yarn lint-language",
+    "lint-fix": "yarn install && yarn lint-md --fix; yarn lint-js --fix; yarn lint-php --fix; echo 'Fixed all auto-fixable problems'",
+    "lint-md": "yarn install && markdownlint-cli2 en/**/*.md",
+    "lint-js": "yarn install && eslint en",
+    "lint-php": "composer install --prefer-dist --no-progress --ansi --no-interaction --no-scripts --no-plugins --optimize-autoloader && vendor/bin/mdphpcs en",
+    "lint-language": "yarn install && vale sync && vale en"
```

Comment:
The individual scripts are used by CI so we can clearly see what is failing and where, without having either a massive hard-to-parse output and without having a failure in one script result in skipping the rest of the linting.

The main `lint` script is useful to run locally to double check all is well before pushing changes.

The `lint-fix` script is useful to run locally to fix anything that _can_ be autofixed, without really caring what script is fixing it.